### DISCUSSION TITLE
An unexpected error occurred: "https://registry.yarnpkg.com/@vercel/next/-/next-3.1.2.tgz: Request failed \"404 Not Found\"".

### DISCUSSION THREAD

**Discussion by giancarlosisasi** [vercel/community#691] [Help] [answered]
Can't deploy/buid on vercel because of this error:

````
Cloning gitlab.com/..., Commit: ***)
--
18:19:33.704 | Cloning completed: 911.265ms
18:19:33.796 | Installing build runtime...
18:19:52.263 | An unexpected error occurred: "https://registry.yarnpkg.com/@vercel/next/-/next-3.1.2.tgz: Request failed \"404 Not Found\"".

````

😢

**Comment by freeman-lab** [comment]
Just saw this very helpful suggestion, can confirm that manually overriding the install command in settings with `npm install` seems to have "fixed" the problem in the repo where we were seeing this. But now wondering if we have to do the same change in all of our repos?

https://twitter.com/poornoodles/status/1542652266204864512?s=20&t=JyZ2dQ93djfxMXuqugArhw

**Comment by swarnava** [comment, marked as answer]
Hi,

We have pushed a fix for this approximately 10 hours ago. Can you try again and let us know how it goes?

**Reply by freeman-lab** [reply]
working correctly for us now!

**Comment by giancarlosisasi** [comment]
@swarnava now it works as expected, thanks!
